subiquity/client/controllers/progress.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, app):
         super().__init__(app)
         self.progress_view = ProgressView(self)
         self.app_state = None
+        self.error_is_reportable: bool = True
```

Comment:
Should this be None instead of True when there is no error?

Reply:
When I added this, my thought process was that the behavior before was effectively if this was always True so I wanted to avoid any issues where ProgressView.update_for_state is called and error_is_reportable wasn't set correctly.

Although now I'm thinking you're right, setting it to None seems better. I think we could change the name to error_is_not_reportable, which will be slightly harder to reason about, but at least None will evaluate to False. What do you think?

Reply:
I went ahead and changed it to `error_is_not_reportable`, let me know what you think.

Reply:
I think that's better yes. May I suggest `has_reportable_error`? Wdyt?

Reply:
That reads much better! I slightly tweaked it to be `has_nonreportable_error` to keep the same truthiness though.

Reply:
Excellent, thanks!